kong/pdk/request.lua

```diff
@@ -820,8 +818,7 @@ local function new(self)
   -- body.age  -- "42"
   function _REQUEST.get_body(mimetype, max_args, max_allowed_file_size)
     check_phase(before_content)
-
-    local content_type = mimetype or _REQUEST.get_header(CONTENT_TYPE)
+    local content_type = mimetype or ngx.var.http_content_type
```

Comment:
"Content-Type" is a built-in single value header, which means `ngx.var.http_content_type` always returns the same value of `get_header("Content-Type")`, and fetching single header by `ngx.var` is quicker than `get_header`(which may use `get_headers()["Content-Type"]` to get the single header).